### ISSUE TITLE
TV shows / Movies: incorrectly adds artists and albums to section

### ISSUE THREAD

**Issue by un1versal** [jez500/chorus2#112]
Navigate to TV shows > All TV shows (Same with Movies > All Movies) and voila question mark city.

![capture](https://cloud.githubusercontent.com/assets/3521959/15056837/a4868490-130b-11e6-9317-9b8e2f277d37.PNG)
```
    Chorus version
    2.0.14
    Kodi version
    17.0
    Remote control
    Remote control is set up correctly
    Local audio
    HTML 5
```

**Comment by un1versal** [comment]
Hers hoping you are OK Jezz.
This is a shameless ping, is development still alive and well?

**Comment by jeremy-doghouse** [comment]
Hey @un1versal 

Yeah still going, just been minimal time in my life atm, lots of other projects on the go, will hopefully get back into it soon.

**Comment by jez500** [comment]
Hey @un1versal 

Yeah still going, just been minimal time in my life atm, lots of other projects on the go, will hopefully get back into it soon.

**Comment by un1versal** [comment]
Good to hear back from you so soon.

Yes, that ol nugget, death taxes and real life... didnt you hear? You're not allowed real life or other projects anymore :grinning: 

cant wait to seee what you add next so I can start complaining about other bugs... hahaha

be well till then.

**Comment by jez500** [comment]
Sorry for the long wait @un1versal - this is now fixed in 2.0.15
